storefront/app/assets/stylesheets/workarea/storefront/base/_images.scss

```diff
@@ -27,3 +27,15 @@ img {
 img[width], img[height] { /* [2] */
     max-width: none;
 }
+
+
+/**
+ * Styles for images while they're lazy loading
+ */
+img.lazy-image {
```

Comment:
Is this necessarily a good idea? Last time I read up, using progressive JPGs was good to allow them to see that images are coming in (also why our jpeg processing converts to progressive).